docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -2963,18 +2963,22 @@ column-definition is:
 <<<
 ```
 data-type is:
-    char[acter] [(length [characters])]
-          [character set char-set-name]
-          [upshift] [[not]casespecific]
+    char[acter] [length [characters]]
+                [character set char-set-name]
+                [upshift] [[not]casespecific]
   | char[acter] varying (length [characters])
-          [character set char-set-name]
-          [upshift] [[not]casespecific]
-  | varchar (length) [character set char-set-name]
+                        [character set char-set-name]
+                        [upshift] [[not]casespecific]
+  | varchar (length [characters]) 
+            [character set char-set-name]
+            [upshift] [[not]casespecific]
+  | varchar2 (length [characters]) 
+             [character set char-set-name]
+             [upshift] [[not]casespecific]          
+  | nchar [length [characters]]

```

Comment:
Should be "nchar (length [characters])"

Reply:
Same as above.
`length [characters]` is optional.

![default](https://user-images.githubusercontent.com/20532956/42792008-e8a9cb4c-89a5-11e8-8066-3a39caa307f3.png)

Reply:
Here should be `nchar [(length) [characters]]` 😁